srml/metadata/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "srml-metadata"
-version = "2.0.0"
+version = "2.0.1"
```

Comment:
No this is not what we requested, you need to alter the structures.

https://github.com/paritytech/substrate/blob/5088c0067e6c21f62c7412ab1d995497121cf595/srml/metadata/src/lib.rs#L315 This needs to be changed to `V5` and `V4` needs to be changed as for the other versions.